etcdctl/README.md

```diff
@@ -69,6 +71,8 @@ The protobuf encoding of the PUT [RPC response][etcdrpc].
 ./etcdctl get foo
 # foo
 # bar
+./etcdctl put foo --ignore-value # this detaches lease
```

Comment:
Yeah i already saw that. The desc : `If ignore_value is set, etcd updates the key using its current value. Returns an error if the key does not exist.` just tells what it does, the doubt after reading this will come to me is why to use it at all when its updating it with old value only, its as good as not updating at all. Don't you feel it sounds that way? Anyways that's just my opinion. @xiang90  @heyitsanthony can take a call on it. Otherwise LGTM.
Thanks!